docs/resources/networking_vip.md

```diff
@@ -9,47 +9,56 @@ Manages a VIP resource within HuaweiCloud. This is an alternative to `huaweiclou
 ## Example Usage
 
 ```hcl
-data "huaweicloud_vpc_subnet" "mynet" {
-  name = "subnet-default"
+resource "huaweicloud_vpc_subnet" "test" {
+  ...
 }
 
-resource "huaweicloud_networking_vip" "myvip" {
-  network_id = data.huaweicloud_vpc_subnet.mynet.id
+resource "huaweicloud_networking_vip" "test" {
+  network_id = data.huaweicloud_vpc_subnet.test.id
```

Comment:
you have updated to use resource instead of data source

Reply:
finished.